data/logs_model/splunk_logs_model.py

```diff
@@ -0,0 +1,85 @@
+import json
+import logging
+
+from datetime import datetime
+
+from data import model
+from data.logs_model.logs_producer import LogProducerProxy, LogSendException
+from data.logs_model.logs_producer.splunk_logs_producer import SplunkLogsProducer
+from data.logs_model.shared import SharedModel
+
+logger = logging.getLogger(__name__)
+
+
+class SplunkLogsModel(SharedModel):
+    """
+    SplunkLogsModel implements model for establishing connection and sending events to Splunk cluster
+    """
+
+    def __init__(self, should_skip_logging=None, producer=None, splunk_config=None):
```

Comment:
Is there a point to setting all these arguments to be `None` by default? I understand that makes sense for `should_skip_logging`, but `producer` and `splunk_config` should never be `None`. In fact if either are `None` an exception will occur.

Reply:
makes sense. I see that elastic search does this too [here](https://github.com/quay/quay/blob/master/data/logs_model/elastic_logs.py#L101-L106)